packages/validator/src/validator.ts

```diff
@@ -159,6 +162,24 @@ export class Validator {
       this.clock.start(this.controller.signal);
       this.chainHeaderTracker.start(this.controller.signal);
 
+      const validatorKeys = this.validatorStore.getValidatorKeys();
+
+      for (const pubkeyHex in validatorKeys) {
+        if (!indicesService.validatorPubKeyExists(pubkeyHex)) {
+          this.logger.info("Validator exists in beacon chain", {
+            validatorIndex: validatorStore.getIndexOfPubkey(pubkeyHex),
```

Comment:
redundant to call this `validatorIndex` here, it should be pretty clear we are talking about the validator index here due to the message itself

```suggestion
            index: validatorStore.getIndexOfPubkey(pubkeyHex),
```

Reply:
I don't have a strong opinion on this, I can see a rationale behind utilizing either terminology, to be clear I was following the format used in [lighthouse's VC](https://github.com/ChainSafe/lodestar/issues/4785#issuecomment-1324702738)

Reply:
I would say adding a prefix to an index property only makes sense if there can be multiple types of indexes in the context of the log message. This is not the case here and why aren't we using the term `validatorPubkey` then as well. Would be in favor of removing the `validator` prefix to improve readablity.

https://github.com/ChainSafe/lodestar/issues/4785#issuecomment-1324702738 also log out the fee recipient first, which is questionable, I would probably not look at their log as a good example of how it should be done.

Reply:
Understood. Good point.